BUILD

```diff
@@ -3961,6 +3961,7 @@ grpc_cc_library(
     language = "c++",
     deps = [
         "gpr_base",
+        "gpr_codegen",
```

Comment:
Is this actually needed?

Reply:
Oh right! It is not required. Removed the target.